### PR TITLE
Added zip task

### PR DESCRIPTION
**Pull request by awolkers**
- added zip task to dist task
- updated readme.md

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var p = require('./package.json');
```
Comment (branneman) [inline config.js]: Could you use a more descriptive variable name like `package`? The file is getting more and more lines and down where the variable is used one doesn't know what `p` means anymore.
Also; you don't need the `.json` extention, but I guess having it there is more obvious. Do you think it should stay?

File: tasks/zip.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var config  = require('../config');
+var gulp    = require('gulp');
+var zip     = require("gulp-zip");
```
Comment (branneman) [inline tasks/zip.js]: Almost this time! ;)

File: readme.md [hunk 3]
```diff
@@ -19,15 +19,23 @@
...
 gulp dist
 ```
+
+## How to: Test
+```
+gulp test
+```
+The test suite contains:
+- JavaScript code linting
+- SASS file code linting
```
Comment (branneman) [inline readme.md]: The official productname for Sass is not all uppercase, let's refer to it correctly, just "Sass".

File: config.js (hunk 1 above)
Comment (awolkers) [inline config.js]: > First, in strict mode a short list of identifiers become reserved keywords. These words are implements, interface, let, package, private, protected, public, static, and yield. In strict mode, then, you can't name or use variables or arguments with these names.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Paving_the_way_for_future_ECMAScript_versions

Comment (branneman) [inline config.js]: How about `pkg`? That's the grunt & gulp convention afaik.

Comment (awolkers) [inline config.js]: ok, will change it to `pkg`

**Pull request merged**